drivers/display/display_sdl.c

```diff
@@ -344,6 +344,19 @@ static int sdl_display_set_contrast(const struct device *dev,
 	return -ENOTSUP;
 }
 
+static int sdl_display_set_scroll_area(const struct device *dev,
+			    uint16_t tfa,
+				uint16_t bfa)
+{
+	return -ENOTSUP;
```

Comment:
Since I have no experience with sdl and it would be a completly different approach I cant promise a working solution for this PR. Perhaps it would be better to add it separately?

Reply:
It no problem.
Please create new issue or PR to remember.